packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -37,7 +37,7 @@ export const tour_step_config: Step[] = [
             </Text>
         ),
 
-        target: '.trading-hub-header__tradinghub--onboarding--logo',
```

Comment:
I cannot find any usage of this file in the code base, seems like a dead code, can you help to remove it totally.

Reply:
ok 📝